src/patternfly/components/MenuToggle/menu-toggle-controls.hbs

```diff
@@ -1,6 +1,13 @@
 <span class="pf-c-menu-toggle__controls{{#if menu-toggle-controls--modifier}} {{menu-toggle-controls--modifier}}{{/if}}"
   {{#if menu-toggle-controls--attribute}}
     {{{menu-toggle-controls--attribute}}}
-  {{/if}}>
+  {{/if}}
+  {{#if menu-toggle--IsDisabled}}
+    disabled
+  {{/if}}
+  {{#if menu-toggle-button--IsDisabled}}
+    disabled
+  {{/if}}
```

Comment:
You shouldn't need this - doesn't look like it's getting styles from the `disabled` attribute, and `disabled` isn't valid on a `<span>`. It's only valid on [form elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled) like inputs, text areas, buttons, etc.

```suggestion
```